ecs_deploy/ecs.py

```diff
@@ -214,7 +214,7 @@ def parse_command(command):
                     "command: {} resulting in error: {}"
                     .format(command, str(e)))
 
-        return command.split(' ')
+        return [cmd for cmd in command.split(" ") if cmd]
```

Comment:
Thanks @normoes !

I was just double checking this. Only removing the explicit empty string (`' '`) will have the same  result, as `split()` then splits "by arbitrary strings of whitespace characters (space, tab, newline, return, formfeed)" ([docs](https://docs.python.org/2/library/string.html#string.split))
 
```python
>>> '  foo  bar '.split()
['foo', 'bar']
```
Do you want to simplify this?

Reply:
@fabfuel Good point. Done.